docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1834 @@
+{
```

Comment:
<p>I don't feel like this needs to be in a function. It's already simple enough and all of the other parameters like <code>decay</code> , <code>s</code> , <code>include</code> , and <code>finite</code> aren't even being used and are distracting to the reader   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='65'/>

Reply:
<p>Correct...I just wanted to keep the general structure of notebook where I can compare the MERLIN-based discords with the ones discovered from matrix profile for different data sets.</p><p> </p><p>So, I can just do a simple for-loop to show how it works. I can also remove function <code>stumpy_discords</code> (please see below) and just simply show how the same range of discords can be discovered from matrix profile.</p><p> </p><p>And, I can get rid of the last case where I test them on data with <code>np.inf</code> value (as I already did that before for a single <code>m</code> ) </p>